Added VPN to remote access page (#29081)
home-assistant/home-assistant.github.io: source/_docs/configuration/remote.markdown

description: "Setting up remote access for Home Assistant."
---

If you're interested in logging in to Home Assistant while away, you'll have to make your instance remotely accessible. Below are a few options to do this.

<div class='note warning'>

Remember to follow the [securing checklist](/docs/configuration/securing/) before doing this.

</div>

## Home Assistant Cloud

Users of <a href="https://www.nabucasa.com">Home Assistant Cloud</a> can use the <a href="https://www.nabucasa.com/config/remote/">Remote UI</a> without requiring any configuration.

A unique remote URL will be generated and given to you along with a certificate so all your traffic to Home Assistant is encrypted automatically.

## VPN

> A secure way to remotely access your Home Assistant is to use a Virtual Private Network (VPN) service such as [Tailscale](https://github.com/hassio-addons/addon-tailscale) or [ZeroTier One](https://github.com/hassio-addons/addon-zerotier). If you're running a supervised or Home Assistant OS, these are available from the Home Assistant Community Add-ons.

Reviewer: ```suggestion
A secure way to remotely access your Home Assistant is to use a Virtual Private Network (VPN) service such as [Tailscale](https://tailscale.com/) or [ZeroTier One](https://www.zerotier.com/).
```
Removing the link to the Community Store because we can't provide support for third-party applications.

> A VPN connection needs to be established before you can connect to your Home Assistant. This is what makes VPN secure however. Bear in mind when using the Home Assistant Companion app such as on a mobile device without this connection your sensors will not update in Home Assistant.

Reviewer: ```suggestion
A VPN connection needs to be established before you can connect to your Home Assistant from outside your local network. The VPN makes this connection secure. When using the Home Assistant Companion app (such as on a mobile device), without this connection, your sensors will not update in Home Assistant.
```
Rephrase is my personal preference. Feel free to ignore.

Reviewer 2: I like that one.
